testsuite/MDAnalysisTests/__init__.py

```diff
@@ -98,7 +98,7 @@
 
 # keep in sync with RELEASE in setup.py
 __version__ = "0.19.1-dev"
-from MDAnalysis import __authors__
+#from MDAnalysis import __authors__
```

Comment:
I think you cannot import MDAnalysis before the actual tests... at least I vaguely remember that we didn't allow it because of coverage.  IIRC, we do not import anything in MDAnalysis except when running tests. On these grounds alone, this line should not be here.

I am not sure why it would break the duecredit tests although I also remember that we're doing some fairly fiddly things to make sure that duecredit is either present or not.

Reply:
Yes, I know, it's not in https://github.com/MDAnalysis/mdanalysis/wiki/UnitTests#requirements-for-tests-in-mdanalysistests ...